src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -99,6 +100,44 @@
           </template>
         </ChiDataTableToolbar>
       </template>
+      <template #bulkActions>
+        <div class="chi-bulk-actions__buttons -pl--3">
```

Comment:
Buttons should go into slot with proper left padding. Setting padding from outside should not be necessary.